src/NumericalAlgorithms/Spectral/SwshCoefficients.hpp

```diff
@@ -8,9 +8,92 @@
 #include <sharp_cxx.h>
 
 #include "NumericalAlgorithms/Spectral/SwshCollocation.hpp"
+#include "Utilities/ForceInline.hpp"
 
 namespace Spectral {
 namespace Swsh {
+
+/// \ingroup SwshGroup
+/// \brief Convenience function for determining the number of spin-weighted
+/// spherical harmonics coefficients that are stored for a given `l_max`
+constexpr SPECTRE_ALWAYS_INLINE size_t
+number_of_swsh_coefficients(const size_t l_max) noexcept {
+  return (l_max + 1) * (l_max + 2) / 2;  // "triangular" representation
+}
+
+/*!
+ * \ingroup SwshGroup
+ * \brief Compute the relative sign change necessary to convert between the
```

Comment:
I'm not really sure what this sign change means.  Could this be explained better?

Suppose sYlm_sharp = sign_convention(s) sYlm_goldberg where "sign_convention" is some complicated function of s which you write down in the docs at the beginning of TransformJob.  Given two values of s, s_1 and s_2, does this function return sign_convention(s_1) * sign_convention(s_2)?
But why doesn't this depend on m and on real/imag like the other sign convention function?

This should point to the docs at the beginning of TransformJob.

Reply:
The sign change is as you suspected, it's just that the dependence on m cancels. It does depend on the real vs imag and depends on whether each spin is around zero. I've updated the comment to indicate these features more clearly.